project13/ViewController.swift

```diff
@@ -4,16 +4,173 @@
 //
 //  Created by Felipe Gil on 2021-07-09.
 //
-
+import CoreImage
 import UIKit
-
-class ViewController: UIViewController {
-
+//TODO
+#warning("Make the Change Filter button change its title to show the name of the currently filter")
+#warning("Try more than 1 slider with different properties")
+class ViewController: UIViewController, UIImagePickerControllerDelegate, UINavigationControllerDelegate {
+    @IBOutlet weak var imageView: UIImageView!
+    @IBOutlet weak var intensity: UISlider!
+    
+    var currentImage: UIImage!
+    var context: CIContext!
```

Comment:
This doesn't need to be an optional forced unwrapped. You can just add an initial value `CIContext()` and it would be the same. If you don't want it to be set when the class initialized, you can just make it lazy:"

```
lazy var context: CIContext = CIContext()
```

This way, `context` will be initialized only when it's needed (i.e. in `applyProcessing()`).